docs/sources/configuration/_index.md

```diff
@@ -505,6 +505,9 @@ engine:
 # When true, allow queries to span multiple tenants.
 # CLI flag: -querier.multi-tenant-queries-enabled
 [multi_tenant_queries_enabled: <boolean> | default = false]
+
+pre_query_url:
```

Comment:
If we decide to be able to configure the timeout for the preQuery call, I'd rather format this config as:
```yaml
coprocessor:
  pre_query:
    [url: <string> | default = ""]
    [timeout: <duration> | default = 2m]
```

Furthermore, this allows us to add new functions (e.g. post_query) easier.

Reply:
nice.